templates/beta-post.adoc

```diff
@@ -0,0 +1,186 @@
+---
+layout: post
+title: "TITLE"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/GITHUB_USERNAME
+author_github: https://github.com/GITHUB_USERNAME
+seo-title: TITLE - makes sure it ends with - OpenLiberty.io
+seo-description: DESCRIPTION
+blog_description: DESCRIPTION
+---
+= TITLE
+AUTHOR_NAME <https://github.com/GITHUB_USERNAME>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+// // // // // // // //
+// Above:
+// Do not insert any blank lines between any of the lines above.
+// Do not remove or edit the variables on the lines beneath the author name.
+//
+// Replace TITLE with the blog post title eg: MicroProfile 3.3 is now available on Open Liberty 20.0.0.4
+// Replace GITHUB_USERNAME with your GitHub username eg: lauracowen
+// Replace DESCRIPTION with a short summary (~60 words) of the release (a more succinct version of the first paragraph of the post).
+// Replace AUTHOR_NAME with your name as you'd like it to be displayed, eg: Laura Cowen
+// // // // // // // //
+
+BETA_RELEASE_SUMMARY
+
+// // // // // // // //
+// Change the RELEASE_SUMMARY to a quick sentence intro. Make sure to keep the blank lines 
+// // // // // // // //
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+* <<allbeta, All Beta Features>>: a new, larger package that contains all Open Liberty beta and GA features and functions.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools (either the Jakarta EE 9 Beta Features package or the All Beta Features package, depending on what you want to try).
+
+If you try either package, <<feedback, let us know what you think>>.
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+The main change visible to developers in the Jakarta EE 9 planned release is the names of packages changing to accomodate the new `jakarta.*` namespace. In this Open Liberty beta, we have more Jakarta EE 9 features with their name change completed. We've also made excellent TCK progress with these features.
+// // // // // // // //
+// ABOVE:
+// Update the second and third sentence appropriately for this release
+// // // // // // // //
+
+This Open Liberty beta introduces the following Jakarta EE 9 features which now possess their all-new Jakarta EE 9 package names:
+
+* ITEM_1
+* ITEM_2
+* ITEM_3
+// // // // // // // //
+// ABOVE:
+// Change the ITEMS above to list the new changes and features in this
+// beta release
+// // // // // // // //
+
+
+These join the Jakarta EE 9 features in link:https://openliberty.io/blog/?search=beta&key=tag[previous Open Liberty betas]:
```

Comment:
For any URLs in the post that start `https://openliberty.io`, that part must be removed and replaced with `{url-prefix}`, as in `link:{url-prefix}/guides/maven-intro.html[Maven]`. Can you add a comment somewhere instructing the author/editor to do that too?

Reply:
Done